examples/db-templates/mongodb-ephemeral-template.json

```diff
@@ -49,7 +49,11 @@
       "spec": {
         "strategy": {
           "type": "Recreate",
-          "resources": {}
+            "resources": {
+		"limits": {
+		    "memory": "${MEMORY_LIMIT}"
```

Comment:
Yep, it isn't needed anymore.  I'll remove these bits from the templates as well.